examples/ssd/README.md

```diff
@@ -34,6 +34,11 @@ Note that this code requires `cv2` module.
 $ python train.py [--model ssd300|ssd512] [--batchsize <batchsize>] [--gpu <gpu>]
 ```
 
+If you want to use multiple GPUs, use `train_mulit.py`.
```

Comment:
`train_multi.py` is right.
(Sorry, I am reading this PR because it is interesting though this is WIP.)

Reply:
Thank you for pointing out. I will remove `[WIP]` after I confirm this script can reproduce the same performance.